umh-core/docs/production/corporate-firewalls.md

```diff
@@ -16,26 +16,3 @@ agent:
 * You're behind a corporate firewall that you trust
 * You cannot add your corporate CA certificate
 * You understand the security implications
-
```

Comment:
Where have those gone ?

Reply:
you only pushed those to staging right? probbaly need to sync back again

Reply:
Proxy configuration restored from staging branch